frame/election-provider-multi-phase/src/unsigned.rs

```diff
@@ -61,8 +61,10 @@ pub type Assignment<T> =
 /// runtime `T`.
 pub type IndexAssignmentOf<T> = sp_npos_elections::IndexAssignmentOf<SolutionOf<T>>;
 
-#[derive(Debug, Eq, PartialEq)]
-pub enum MinerError {
+pub type SolverErrorOf<T> = <<T as Config>::Solver as NposSolver>::Error;
+
+#[derive(frame_support::DebugNoBound, frame_support::PartialEqNoBound)]
+pub enum MinerError<T: Config> {
```

Comment:
and here, which is not related to this PR but you can fix it 🙈  in general all pub items should be documented